src/client/src/app/components/join-game/join-game.component.ts

```diff
@@ -7,14 +7,22 @@ import { SocketService } from 'src/app/services/socket.service';
   templateUrl: './join-game.component.html',
   styleUrls: ['./join-game.component.scss']
 })
+
 export class JoinGameComponent implements OnInit {
+
+  errMsg: string | null= null;
+
     constructor(private socket:Socket) { }
 
   ngOnInit(): void {
   }
 
   addPlayer(name:string){
-    this.socket.emit("add-name", name)
+    this.socket.emit("add-name", name);
+    this.socket.on("message", (data:string)=>{
```

Comment:
replace event name **message** with **error-message** to avoid clashing with other event whose name is also message